core/domain/base_model_validators_test.py

```diff
@@ -84,7 +84,9 @@ class MockSnapshotMetadataModelValidator(
     def _get_external_id_relationships(cls, item):
         return [
             base_model_validators.ExternalModelFetcherDetails(
-                'external_model_ids', MockModel, [])]
+                'external_model_ids', MockModel, []),
+            base_model_validators.ExternalModelFetcherDetails(
+                'committer_ids', MockModel, ['OppiaMigrationBot', 'User-1'])]
```

Comment:
Add tests for both `allow_system_user_ids` and `allow_pseudonymous_ids` set to `True`.

Reply:
Done.